02_configure_host.sh

```diff
@@ -9,7 +9,7 @@ source common.sh
 # Note we copy the playbook so the roles/modules from tripleo-quickstart
 # are found without a special ansible.cfg
 export ANSIBLE_LIBRARY=./library
-ansible-playbook -e roles_path=$PWD/roles -b -vvv tripleo-quickstart-config/metalkube-setup-playbook.yml -e @tripleo-quickstart-config/metalkube-nodes.yml -e local_working_dir=$HOME/.quickstart -e virthost=$HOSTNAME -e @config/environments/dev_privileged_libvirt.yml -i tripleo-quickstart-config/metalkube-inventory.ini
+ansible-playbook -e non_root_user=$USER -e roles_path=$PWD/roles -b -vvv tripleo-quickstart-config/metalkube-setup-playbook.yml -e @tripleo-quickstart-config/metalkube-nodes.yml -e local_working_dir=$HOME/.quickstart -e virthost=$HOSTNAME -e @config/environments/dev_privileged_libvirt.yml -i tripleo-quickstart-config/metalkube-inventory.ini
```

Comment:
```suggestion
ansible-playbook -e "non_root_user=$USER" -e "roles_path=${PWD}/roles" -b -vvv tripleo-quickstart-config/metalkube-setup-playbook.yml -e @tripleo-quickstart-config/metalkube-nodes.yml -e "local_working_dir=${HOME}/.quickstart" -e "virthost=$HOSTNAME" -e @config/environments/dev_privileged_libvirt.yml -i tripleo-quickstart-config/metalkube-inventory.ini
```
Since you are touching the line, let's make it safer bash

Reply:
yeah, I'm also going to remove the hardcoded working_dir in the ansible playbooks and udpate the command in libvirt_cleanup.sh